components/_global-footer.scss

```diff
@@ -25,9 +25,10 @@
 	$mobile-row-header-international-padding-bottom: 35px;
 
 	$section-line-width: 4px;
-	$desktop-row-top-height: 205px + $section-line-width;
+	$desktop-row-top-height: 206px + $section-line-width;
 
 	// others
+	$footer-line-height: 1.3em;
```

Comment:
While we're introducing new `line-height` value do you think we should use it for entire `global-footer`? I see old line height used in `&__bottom-bar` section